templates/default/sssd.conf.erb

```diff
@@ -133,4 +133,13 @@ ldap_group_member = msSFU30PosixMember
 <% else -%>
 ldap_group_member = uniqueMember
 <% end -%>
+<% if !node[:authconfig][:access_provider].nil? -%>
```

Comment:
cleaner here to use unless node[:authconfig][:access_provider].nil?, instead of "if !"